### PR TITLE
backport mojin changes for next research batch

### PR DESCRIPTION
**Pull request by fmessmer**
ref https://github.com/mojin-robotics/cob4/issues/2492

backporting some changes from mojin-robotics that might be helpful for setting up the next research robot batch
changes are base on melding the two repos....then copy-paste...

@pgehring FYI

### CODE REVIEW

File: PostInstallCob4.sh [hunk 1]
```diff
@@ -420,10 +479,10 @@ else
...
 fi
 
-if [ ! -d /u/robot/git/setup_cob4 ]; then
-  echo -e "${blue} clone ipa320/setup_cob4...${NC}"
-  mkdir /u/robot/git
-  git clone git@github.com/ipa320/setup_cob4 /u/robot/git/setup_cob4
+if [ ! -d $home_prefix/robot/git/setup_cob4 ]; then
+  echo -e "${blue} clone mojin-robotics/setup_cob4...${NC}"
+  mkdir $home_prefix/robot/git
+  git clone git@github.com/mojin-robotics/setup_cob4 $home_prefix/robot/git/setup_cob4
```
Comment (floweisshardt) [inline PostInstallCob4.sh:485]: should still be ipa320, right?

**Comment by fmessmer** [comment]
re-trigger GHA

File: PostInstallCob4.sh (hunk 1 above)
Comment (fmessmer) [reply PostInstallCob4.sh]: yes, slipped through, done

**Comment by fmessmer** [comment]
this backport does not yet include:
- https://github.com/mojin-robotics/setup_cob4/pull/392
- https://github.com/mojin-robotics/setup_cob4/pull/391

**Pull request merged**